--- /data/openpilot/selfdrive/car/toyota/carcontroller.py
+++ tmp/carcontroller.py.patched
@@ -86,4 +86,4 @@
     # on entering standstill, send standstill request
-    if CS.out.standstill and not self.last_standstill and CS.CP.carFingerprint not in NO_STOP_TIMER_CAR:
-      self.standstill_req = True
+    #if CS.out.standstill and not self.last_standstill and CS.CP.carFingerprint not in NO_STOP_TIMER_CAR:
+    #  self.standstill_req = True
     if CS.pcm_acc_status != 8:
