--- /data/openpilot/selfdrive/monitoring/driver_monitor.py	2022-02-18 14:55:59.000000000 +0800
+++ tmp/driver_monitor.py.patched	2022-02-18 14:54:08.000000000 +0800
@@ -20,6 +20,6 @@
     self._DT_DMON = DT_DMON
-    self._AWARENESS_TIME = 35. # passive wheeltouch total timeout
+    self._AWARENESS_TIME = 35000. # passive wheeltouch total timeout
     self._AWARENESS_PRE_TIME_TILL_TERMINAL = 12.
     self._AWARENESS_PROMPT_TIME_TILL_TERMINAL = 6.
-    self._DISTRACTED_TIME = 11. # active monitoring total timeout
+    self._DISTRACTED_TIME = 11000. # active monitoring total timeout
     self._DISTRACTED_PRE_TIME_TILL_TERMINAL = 8.
